packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -1,467 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Thanks @vpomerleau.  I checked the output of your commit in combination with the associated [workflow update ](https://github.com/mozilla/fxa-content-server-l10n/pull/657) on `fxa-content-server-l10n` with moz-l10n-lint / compare-locales and not seeing any issues. Did a quick spot check manually as well but nothing stands out. I'll approve the review.

@LZoog  - Before merging these changes, we'll need to coordinate a time once the final PRs are ready. We'll use those PRs to do a dry run migration for Pontoon.

Reply:
@vpomerleau @LZoog Do we happen to have the list of unused strings available somewhere still?

Reply:
@StaberindeZA Here's a copy, please expand:

<details>
<summary>Unused Strings</summary>

```
# UNUSED STRINGS?

# # $productName (String) - The name of the subscribed product.
# # $amount (Number) - The amount billed. It will be formatted as currency.

# # $intervalCount (Number) - The interval between payments, in days.
# day-based-plan-details-amount = { $intervalCount ->
#   [one] { $productName } billed { $amount } daily
#   *[other] { $productName } billed { $amount } every { $intervalCount } days
# }
# # $intervalCount (Number) - The interval between payments, in weeks.
# week-based-plan-details-amount = { $intervalCount ->
#   [one] { $productName } billed { $amount } weekly
#   *[other] { $productName } billed { $amount } every { $intervalCount } weeks
# }
# # $intervalCount (Number) - The interval between payments, in months.
# month-based-plan-details-amount = { $intervalCount ->
#   [one] { $productName } billed { $amount } monthly
#   *[other] { $productName } billed { $amount } every { $intervalCount } months
# }
# # $intervalCount (Number) - The interval between payments, in years.
# year-based-plan-details-amount = { $intervalCount ->
#   [one] { $productName } billed { $amount } yearly
#   *[other] { $productName } billed { $amount } every { $intervalCount } years
# }
# ## Payment form

# payment-ccn =
#   .label = Card number
# payment-exp =
#   .label = Expiration
# payment-cvc =
#   .label = CVC
# payment-zip =
#   .label = ZIP code
# payment-validate-zip-required = Zip code is required
# payment-validate-zip-short = Zip code is too short
# ## Subscription redirect

# sub-redirect-ready = Your subscription is ready
# sub-redirect-copy = Please take a moment to tell us about your experience.
# sub-redirect-skip-survey = No thanks, just take me to my product.
# ## Subscription upgrade

# sub-update-card-exp = Expires { $cardExpMonth }/{ $cardExpYear }
# sub-change-indicator =
#   .aria-label = change indicator
# ##

# account-activated = Your account is activated, <userEl/>
# ## Payment confirmation

# payment-confirmation-alert = Click here to download
# payment-confirmation-mobile-alert = Didn’t open app? <a>Click Here</a>
# payment-confirmation-billing-heading = Billed to
# ## Coupon
# coupon-discount = Discount
# coupon-discount-applied = Discount Reward Applied
# # Subscriptions - Reactivate
# reactivate-panel-date = You cancelled your subscription on { $date }.
# ## Subscription item

# sub-item-missing = Problem loading subscriptions
# sub-item-missing-msg = Please try again later.
# ## Subscriptions error
# sub-subscription-error =
#   .title = Problem loading subscriptions
# # Routes
# pay-with-heading-other = Select payment option
```
</details>